src/data-requests/frozen-incidence.ts

```diff
@@ -39,30 +39,34 @@ export async function getFrozenIncidenceHistory(
     .replace(date_pattern, "$3-$2-$1");
   const lastUpdate = new Date(dateString);
 
-  let districts = json.map((district) => {
-    const name = district["LK"];
-    const ags = district["LKNR"].toString().padStart(5, "0");
+  let districts = json
+    .filter((district) => !!district["NR"])
+    .map((district) => {
+      console.log(district);
+      const name = district["LK"];
+      const ags = district["LKNR"].toString().padStart(5, "0");
 
-    let history = [];
+      let history = [];
 
-    // get all date keys
-    const dateKeys = Object.keys(district);
-    // ignore the first three elements (rowNumber, LK, LKNR)
-    dateKeys.splice(0, 3);
-    dateKeys.forEach((dateKey) => {
-      const date = new Date(
-        dateKey.toString().replace(date_pattern, "$3-$2-$1")
-      );
-      history.push({ weekIncidence: district[dateKey], date });
-    });
+      // get all date keys
+      const dateKeys = Object.keys(district);
+      // ignore the first three elements (rowNumber, LK, LKNR)
+      dateKeys.splice(0, 3);
+      dateKeys.forEach((dateKey) => {
+        const date = AddDaysToDate(
```

Comment:
using function "AddDaysToDate" without import!

Reply:
Good spot! I removed that part now, since it wasn't present in `v2`. In my local fork I shift the dates of the frozen incidences by one, so that they line up with the other data.

Reply:
same as i do, wrote a PR many days ago, but Marlon is not merging this .....